projects/cashmere/src/lib/sass/form-field.scss

```diff
@@ -145,6 +145,10 @@ $wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
     padding: $infix-padding 10px;
 }
 
+@mixin hc-form-field-infix-mobile() {
+    font-size: calculateRem(16px);
```

Comment:
I was a bit confused when I first saw `calculateRem()`. I've been using this mixin everywhere for font size rems: 
`@include fontSize(16px)`. 

Turns out fontSize() mixin just uses calculateRem() under the covers. Not totally sure why we two versions in the library to do the same thing.

So I guess nothing really needs to be changed here... but should figure out why both exist and why we'd use one over the other.

Reply:
Hah, good point - I've mostly used `fontSize` as well.  In this case I just did a copy/paste from what was being used elsewhere in this stylesheet.  But pretty sure those two functions are totally redundant like you said.